.gitlab.mk

```diff
@@ -108,21 +120,26 @@ package: deploy_prepare
 	TARBALL_EXTRA_ARGS="--exclude=*.exe --exclude=*.dll"                                                            \
 	PRESERVE_ENVVARS="TARBALL_EXTRA_ARGS,${PRESERVE_ENVVARS}" ./packpack/packpack
 
-# found that libcreaterepo_c.so installed in local lib path
-deploy: export LD_LIBRARY_PATH=/usr/local/lib
-
 deploy:
-	echo ${GPG_SECRET_KEY} | base64 -d | gpg --batch --import || true
-	case "${GITHUB_REF}" in                                       \
-	refs/tags/*-alpha*|refs/tags/*-beta*|refs/tags/*-rc*) \
-	    ./tools/update_repo.sh -o=${OS} -d=${DIST}            \
-			-b="${PRERELEASE_REPO_S3_DIR}/${BUCKET}" build ; \
-	        ;;                                                    \
-	refs/tags/*)                                                  \
-		./tools/update_repo.sh -o=${OS} -d=${DIST}            \
-			-b="${RELEASE_REPO_S3_DIR}/${BUCKET}" build ; \
-	        ;;                                                    \
-	esac
+	if [ -z "${REPO_TYPE}" ]; then \
+		echo "Env variable 'REPO_TYPE' must be defined!"; \
+		exit 1; \
+	fi; \
+	CURL_CMD="curl \
+		--location \
+		--fail \
+		--silent \
+		--show-error \
+		--retry 5 \
+		--retry-delay 5 \
+		--request PUT ${RWS_ENDPOINT} \
+		--user $${RWS_AUTH} \
+		--form product=${PRODUCT_NAME}"; \
```

Comment:
`product` is needed for deb packages. Does it work with rpm?

Reply:
Yes, it works. In the case of RPM, RWS just ignores it.

Reply:
ok. thanks